config/initializers/highlight_override.rb

```diff
@@ -90,9 +99,23 @@ def query(user_uuid:)
       end
 
       highlights.sort_by!{ |highlight| [source_id_order[highlight.source_id], highlight.order_in_source] }
+    else
+      # Have to sort by something for pagination to be sensible, choose created_at
+      highlights.sort_by!(&:created_at)
```

Comment:
shouldnt the highlights be sorted by their linked list order?

Reply:
No because without source IDs specified, the pages are going to be all mixed up anyway.